WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java

```diff
@@ -141,6 +172,37 @@ public void onActivityCreated(Bundle savedInstanceState) {
         AnalyticsTracker.flush();
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onAccountChanged(AccountStore.OnAccountChanged event) {
+        if (!isAdded()) {
```

Comment:
Not sure if this points to a problem, but i noticed this is triggered whenever I close app settings, even when I haven't made any changes.

Reply:
This is called because there is a call to `FETCH_ACCOUNT` somewhere else in the code. 
Will add a guard to that checks  `event.causeOfChange == AccountAction.FETCH_SETTINGS` before doing anything else.

Reply:
It `WPMainActivity` that calls for an update
```
        // Update account to update the notification unseen status
        if (mAccountStore.hasAccessToken()) {
            mDispatcher.dispatch(AccountActionBuilder.newFetchAccountAction());
        }
```